firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -115,6 +121,7 @@ class Setting: NSObject {
 
     init(
         title: NSAttributedString? = nil,
+        centeredTitle: NSAttributedString? = nil,
```

Comment:
Do we still need the `centeredTitle` here or could we use the `title` as well now that there are two different cell classes?